app/components/admin/menu_component.rb

```diff
@@ -54,15 +54,14 @@ def profiles?
     end
 
     def settings?
-      controllers_names = ["settings", "tenants", "tags", "geozones", "images",
-                           "content_blocks", "local_census_records", "imports"]
+      controllers_names = ["settings", "tenants", "tags", "geozones", "local_census_records", "imports"]
       controllers_names.include?(controller_name) &&
         controller.class.module_parent != Admin::Poll::Questions::Answers
```

Comment:
Hi @dnsos,

Reviewing this PR, I think this line was added to allow for the "images" value in the controllers_names.

Could it be that by moving this "images" to another method, this line can be removed?

Reply:
Hi @taitus, thanks for the hint, I wasn't aware of that special case.

After removing the line I have tested the app manually and there is indeed no problem. I've tried to explain this to the best of my knowledge in [809875e](https://github.com/consuldemocracy/consuldemocracy/pull/5251/commits/809875ed357b95f9edddf881dd3439b70f53e625).

 A spec to verify that the line is unnecessary could look something like that (note the used controller and the text to match):

```rb
# spec/components/admin/menu_component_spec.rb
require "rails_helper"

describe Admin::MenuComponent, :admin, controller: Admin::Poll::Questions::Answers::ImagesController do
  let(:component) { Admin::MenuComponent.new }

  it "highlights currently active list item" do
    render_inline component
    expect(page).to have_css "[aria-current='true'] a", exact_text: "Polls"
  end
end
```

It feels a little overspecific though, given that the menu component has a lot of other cases that wouldn't be covered by such a minimal test. Happy to receive directions on whether such a test would be valuable.

Reply:
Hi @dnsos 

Thanks for adding the changes. The new commit added is just what was missing. 👏 

In my opinion the test you suggest is not necessary to add for now. Since we don't seem to have any test for that component and it's a bit weird that there is only one and it's at that level of detail. 🤔 

What you could do if you don't mind would be to rebase the `master` branch and remove the commit `Merge branch 'consuldemocracy:master' into intuitive-admin-menu-sections`. I think that this way it will be ready for the merge.

Thank you very much!

Reply:
Hey @taitus thanks for the reply and I agree on your opinion about the test.

The merge commit is now gone!